cockatrice/src/carddbparser/cockatricexml3.cpp

```diff
@@ -77,6 +77,7 @@ void CockatriceXml3Parser::loadSetsFromXml(QXmlStreamReader &xml)
         if (name == "set") {
             QString shortName, longName, setType;
             QDate releaseDate;
+            short priority;
```

Comment:
I think we can leave the old v3 parser as is

Reply:
Reverted.